.github/workflows/release-freeze.yml

```diff
@@ -10,121 +10,17 @@ on:
         - major
         - minor
         - pre_release
+      mcore_version:
+        description: 'Version of MCore to use (must be a valid git ref)'
```

Comment:
should this say aligner?

Reply:
Ah, `mcore_version` is used in NeMo to override the mcore ARG in `Dockerfile.ci` to the release candidate of mcore (instead a sha). Shall we do something similar here?